migrations/sql/afterMigrate__1__seed_data.sql

```diff
@@ -393,13 +393,15 @@ INSERT INTO mine_incident_followup_investigation_type
 	mine_incident_followup_investigation_type_code,
 	description,
 	display_order,
+    active_ind,
 	create_user,
 	update_user
 	)
 VALUES
-	('MIU', 'Yes - MIU Investigation', 10, 'system-mds', 'system-mds'),
-	('INS', 'Yes - Inspector Investigation', 20, 'system-mds', 'system-mds'),
-	('NO', 'No', 30, 'system-mds', 'system-mds')
+	('MIU', 'Yes - MIU Investigation', 10, 'T', 'system-mds', 'system-mds'),
+	('INS', 'Yes - Inspector Investigation', 20, 'T', 'system-mds', 'system-mds'),
+	('NO', 'No', 30, 'T', 'system-mds', 'system-mds'),
+    ('HUK', 'Historical - Unknown', 40, 'F', 'system-mds', 'system-mds')
```

Comment:
The front-end needs to be modified to not display this in the list of options.

Reply:
does it? Aaron is fine if for this week we allow it to be a fully fledged option with no special behaviour, there will be a ticket to lock it down.

Reply:
It should be a one line change, .filter((x) => x.active_ind=true). Might be more work due to overhead to put it off, but if you want to separate it into another ticket, fine.